dist/formats/case_study/publisher_v2/links.json

```diff
@@ -0,0 +1,138 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "previous_version": {
+      "type": "string"
+    },
+    "base_path": {
```

Comment:
You're right, I've mistakenly combined `v2_metadata.json` when building the links schema. Will fix :+1: